frontend/src/components/SignOutButton.tsx

```diff
@@ -18,6 +18,8 @@ export const SignOutButton = () => {
   return (
     <Button
       variant="outlined"
+      size="large"
+      color="secondary"
       className="ml-auto"
       onClick={() => handleLogout(instance)}
     >
```

Comment:
Don't worry too much about the sign out button, I think we'll put that in the nav bar in the chats page